test/remote/gateways/remote_fusebox_test.rb

```diff
@@ -46,6 +46,19 @@ def test_unsuccessful_authorize
     assert_match(/^0041 BAD ACCT NUMBER/, response.message)
   end
 
+  def test_successful_auth_and_reverse
+    @reference = unique_reference
+    assert response = @gateway.store_and_auth(@credit_card, reference: @reference)
+    assert_success response
+    assert_match(/^0000, COMPLETE/, response.message)
+    @token = response.params['token']
+
+    assert response = @gateway.auth_reversal(100, @token, reference: @reference)
```

Comment:
Here is where I tested (in a remote test, too) that you can reverse auth with the token and not the PAN. 

The fusebox API (and my activemerchant library) both accept a token anywhere they take a PAN.